.tekton/konflux-testing-build.yaml

```diff
@@ -9,7 +9,16 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "500"
     # TODO(ROX-21073): re-enable for all PR branches
-    pipelinesascode.tekton.dev/on-cel-expression: (event == "push" && target_branch == "master") || (event == "pull_request" && (source_branch.contains("rhtap") || source_branch.contains("konflux")))
+    pipelinesascode.tekton.dev/on-cel-expression:
+      (
+        event == "push" && target_branch.matches("refs/tags/*-nightly-*|master")
```

Comment:
Btw, `"refs/tags/*-nightly-*|master"` should be a valid regex; `match()` expects regex not globs.